app/components/download-switch/download-switch.tsx

```diff
@@ -314,14 +322,16 @@ export function DownloadSwitch({
     }
     __DEV__ && console.log("Determining download status.", tracks, switchStatus, statuses)
     determineDownloadStatuses()
-    return () => (isMounted = false)
+    return () => {
+      isMounted = false
+    }
```

Comment:
TS complained about the callback of `useEffect` being a function which returns `() => boolean`, but it only accepts `() => void`.

Reply:
I like how thorough TS is!